### PR TITLE
2016 02 15 np move topic page

### PR DESCRIPTION
**Pull request by np**
I will rebase it onto master.

### CODE REVIEW

**Comment by Mikolaj** [comment]
LGTM. Better to rebase soon --- such commits get into conflicts a lot.

File: src/Frontend/Topics.hs [hunk 1]
```diff
@@ -0,0 +1,137 @@
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE FlexibleInstances   #-}
+{-# LANGUAGE KindSignatures      #-}
+{-# LANGUAGE LambdaCase          #-}
+{-# LANGUAGE OverloadedStrings   #-}
+
+{-# OPTIONS_GHC -Werror -Wall #-}
+
+module Frontend.Topics
```
Comment (andorp) [inline src/Frontend/Topics.hs]: What if we move it to `Frontend.Page.Topics` or should I move `Frontend.Page.CreateIdea` to `Frontend.CreateIdea`? I prefer collect every page under the Page submodule.

Comment (fisx) [inline src/Frontend/Topics.hs]: > What if we move it to `Frontend.Page.Topics` or should I move `Frontend.Page.CreateIdea` to `Frontend.CreateIdea`? I prefer collect every page under the Page submodule.

Yes, i would also like this to go to `Frontend.Page.Topics`.

Comment (np) [inline src/Frontend/Topics.hs]: I don't mind but can we do this later?

Comment (andorp) [inline src/Frontend/Topics.hs]: I can do this. My new change will depend on it.

**Pull request merged**